robot/lib/Inc/drivers/KickerBoard.hpp

```diff
@@ -132,17 +132,53 @@ class KickerBoard : public AVR910 {
 private:
     bool verbose;
 
+    /**
+     * Voltage threshold for kicker board to be considered charged (voltage mapped to 0-255)
+     */
     const uint8_t isChargedCutoff = 230;
 
+    /**
+     * Stores whether the breakbeam has been tripped
+     */
     bool _ball_sensed = false;
+
+    /**
+     * Stores whether Kicker is responding back with voltage values
```

Comment:
I would be more specific and say
```suggestion
     * Stores whether Kicker is responding back with non-zero voltage values
```